packages/pwa/config/default.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
I'm a bit confused by the name of this file, I thought the name of the config is the mrt environment id or `local`. When is this default get used?

Reply:
after reading thru the PR i think i get it now.